changelog.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## x.x.x
```

Comment:
This is backwards compatible but it is delete and creating new resources for scheduling. This is why im on the fence on if this should be a minor or major upgrade.

Reply:
Yeah I am not sure what is technically correct, but because it's creating a brand new resource type I would lean towards major.

Reply:
agreed. 8.0.0 it is!